crud/select/compat/select.lua

```diff
@@ -75,6 +75,10 @@ local function build_select_iterator(space_name, user_conditions, opts)
         end
     end
 
+    if opts.first ~= nil then
+        opts.first = math.abs(opts.first)
```

Comment:
It's a side-effect. You modifies user's variable. Probably it's better to save "first" into separate variable and work with it.